src/config.ts

```diff
@@ -53,6 +55,7 @@ const prodConfig = {
   glifNodeUrl: process.env.NEXT_PUBLIC_GLIF_URL ?? '',
   isTestnet: process.env.NEXT_PUBLIC_IS_TESTNET ?? 'false',
   gaTrackingId: process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS_TRACKING_ID ?? '',
+  filfoxUrl: process.env.NEXT_PUBLIC_FILFOX_URL ?? 'https://filfox.info/en',
```

Comment:
Do we need to set NEXT_PUBLIC_FILFOX_URL in deployments or will it work fine without it thanks to this dev/prod split.